README.md

```diff
@@ -250,6 +250,39 @@ The name of your error document inside your `distributionFolder`. This is the fi
 
 ---
 
+**objectHeaders** 
+
+_optional_, no default
+
+```yaml
+custom:
+  client:
```

Comment:
```suggestion
  fullstack:
```

It should be `fullstack` not `client`, right?

Reply:
Yes my bad 🤦‍♂️